pubsub/awssnssqs/awssnssqs.go

```diff
@@ -26,6 +26,13 @@
 // see URLOpener.
 // See https://godoc.org/gocloud.dev#hdr-URLs for background information.
 //
+// Message Delivery Semantics
+//
+// AWS SNS and SQS combine to support at-least-once semantics; applications must
+// call Message.Ack/Nack after processing a message, or it will be redelivered.
```

Comment:
Well, if you call `Nack`, it _will_ be redelivered. So maybe reword.

Reply:
Right, done. I added the reference to Nack later. I removed it again, I don't think it's necessary to mention Nack here.